### PR TITLE
Modified booking with latest data

### PR DESCRIPTION
**Pull request by Sachin796**
Please check, its the skeleton for user and owner booking with all latest data.

### CODE REVIEW

**Comment by Sachin796** [comment]
/user/booking
/owner/booking

File: client/src/components/user/tableHeader/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      <thead>
+        <tr>
+          <th scope="col">Workspace</th>
+          <th scope="col">Start Date</th>
+          <th scope="col">End Date</th>
+          <th scope="col">Price</th>
+          <th scope="col">Action</th>
+        </tr>
+      </thead>
+      <Route exact path="/user/booking" component={createContent} />
```
Comment (jt0398) [inline client/src/components/user/tableHeader/index.js]: This should be {this.state.bookings.map(booking=> {
         return <BookingCard key={booking.id} {...booking} />;
       })}

Please review activity src-29-Stu_FriendRefactor

File: client/src/components/user/tableHeader/index.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./style.css";
+import createContent from "../tableContent";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+function createHeader(props) {
+  return (
+    <table className="table bookingtable">
```
Comment (jt0398) [inline client/src/components/user/tableHeader/index.js]: Tables are not mobile responsive. Please try using Container, Row, and Cols. For the Cols, you can set the xs=12 md=2 when using the React-Bootstrap

File: client/src/components/owner/tableHeader/index.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      <thead>
+        <tr>
+          <th scope="col">Workspace</th>
+          <th scope="col">Start Date</th>
+          <th scope="col">End Date</th>
+          <th scope="col">Price</th>
+          <th scope="col">User ID</th>
+        </tr>
+      </thead>
+      <Route exact path="/owner/booking" component={createOwnerContent} />
```
Comment (jt0398) [inline client/src/components/owner/tableHeader/index.js]: This should be {this.state.bookings.map(booking=> {
return <BookingCard key={booking.id} {...booking} />;
})}

Please review activity src-29-Stu_FriendRefactor

File: client/src/components/owner/tableHeader/index.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import "./style.css";
+import createOwnerContent from "../tableContent";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+function createOwnerHeader(props) {
+  return (
+    <table className="table bookingtable">
```
Comment (jt0398) [inline client/src/components/owner/tableHeader/index.js]: Tables are not mobile responsive. Please try using Container, Row, and Cols. For the Cols, you can set the xs=12 md=2 when using the React-Bootstrap

**Comment by jt0398** [comment]
Old PR

**Pull request closed without merging**